app/views/settings/preferences/show.html.haml

```diff
@@ -7,6 +7,12 @@
   .fields-group
     = f.input :locale, collection: I18n.available_locales, wrapper: :with_label, include_blank: false, label_method: lambda { |locale| human_locale(locale) }
 
+    = f.input :setting_site_theme,
+      collection: ['', 'kkt'],
```

Comment:
not a fan of hardcoding these in here, though I don't know how this is usually done in Rails